x-pack/metricbeat/module/azure/monitor/_meta/data.json

```diff
@@ -0,0 +1,19 @@
+{
+    "@timestamp":"2016-05-23T08:05:34.853Z",
+    "beat":{
+        "hostname":"beathost",
+        "name":"beathost"
+    },
+    "metricset":{
+        "host":"localhost",
+        "module":"azure",
+        "name":"monitor",
+        "rtt":44269
+    },
+    "x-pack/azure":{
```

Comment:
Was this generated? Where is the x-pack prefix coming from?

Reply:
played with the naming and forgot to regenerate, all fixed now